packages/utils/src/lib/filter-by-slug.ts

```diff
@@ -0,0 +1,39 @@
+import { Audit, Group } from '@code-pushup/models';
+import { toArray } from './transform';
+
+export function filterByAuditSlug(
```

Comment:
It would be helpful to distinguish the name of these functions based on what the arrays contain.
```suggestion
export function filterGroupsByAuditSlug(
```